checks/raw/security_policy.go

```diff
@@ -85,8 +108,10 @@ var isSecurityPolicyFile fileparser.DoWhileTrueOnFilename = func(name string, ar
 	}
 	if isSecurityPolicyFilename(name) {
 		tempPath := name
+		// TODO: really should be FileTypeText (.md, .adoc, etc)
```

Comment:
Good catch. Let's use FileTypeText instead.

Reply:
Will do... I was not sure what other problems might be causes by changing the file type (I don't know all the dependencies)

Reply:
Done.